handbook/customers.md

```diff
@@ -1,63 +1,63 @@
 
-L3: In these communications to the customer remember to approach the situation with empathy. Anything could have happened.
+L3: In these communications to the customer, remember to approach the situation with empathy. Anything could have happened.
 
-L4: Be ready to supply an alternative date and time to reschedule the call. Suggest 2-3 options for the customer to choose from.
+L4: Be ready to supply an alternative date and time to reschedule the call. Suggest two to three options from which the customer can choose.
 
 L5: Ensure that contact information is accurate and that meeting invites can be received and accessed by the customer.
 
-L6: Repeat back to the customer the newly agreed upon date and time, as well as the contact information.
+L6: Repeat back to the customer the newly agreed-upon date and time, and the contact information.
 
 L7: Congratulations, you’re ready to set up a new call.
 
-### Next steps after a customer conversation
-After a customer conversation, it can sometimes feel like there are 1001 things to do, but it can be hard to know where to start.  Here are some tips:
+### Next steps after a customer conversation 
+After a customer conversation, it can sometimes feel like there are 1,001 things to do, but it can be hard to know where to start.  Here are some tips: 
 
 
 ## Support process
 
 This section outlines the customer and community support process at Fleet.
 
-- L1: Basic help desk resolution and service delivery -> CS team handles these with occasional support from L2
-- L2: In-depth technical suppport -> CS team with L2 oncall technician
-- L3: Expert product and service support -> CS team liases with L2 and L3 oncall technician.
+- L1: Basic help desk resolution and service delivery -> CS team handles these with occasional support from L2 
+- L2: In-depth technical support -> CS team with L2 onc-all technician 
+- L3: Expert product and service support -> CS team liases with L2 and L3 on-call technician
 
-In each case, if possible, the resulting solution should be made more clear in the documentation and/or the FAQs.
+In each case, if possible, the resulting solution should be made clearer in the documentation and/or the FAQs.
 
 The support process is accomplished via on-call rotation and the weekly on-call retro meeting.
 
-The on-call engineer is responsible for responding to technical Slack comments, Slack threads, and GitHub issues raised by customers and the community which cannot handled by the Customer Success team.
+The on-call engineer is responsible for responding to technical Slack comments, Slack threads, and GitHub issues raised by customers and the community that the Customer Success team cannot handle.
 
-Slack messages have a 24 hour SLA and the Slack channel should have a notice at the top explaining so.
+Slack messages have a 24-hour SLA, and the Slack channel should have a notice at the top explaining so.
```

Comment:
@tgauda, in reference to "SLA":
For easy reading and translation, please spell things out and avoid acronyms.